core/templates/dev/head/forms/formBuilder.js

```diff
@@ -559,6 +561,9 @@ oppia.factory('rteHelperService', [
           RTE_COMPONENT_SPECS[componentId].customization_arg_specs),
         name: RTE_COMPONENT_SPECS[componentId].frontend_name,
         iconDataUrl: RTE_COMPONENT_SPECS[componentId].icon_data_url,
+        previewDataUrl: RTE_COMPONENT_SPECS[componentId].preview_data_url,
```

Comment:
Is it possible to just have a single previewUrlTemplate key? If it's a data URL then nothing gets interpolated (since dataURLs don't have <[...]> characters), and otherwise it gets interpolated as usual.